tests/lean/bad_unification_hint.lean.expected.out

```diff
@@ -1,4 +1,4 @@
-bad_unification_hint.lean:7:0: warning: declaration 'append_cons_hint' uses sorry
 bad_unification_hint.lean:7:0: error: invalid unification hint, failed to unify pattern after unifying constraints
-bad_unification_hint.lean:12:0: warning: declaration 'cons_append_hint'' uses sorry
+bad_unification_hint.lean:7:0: warning: declaration 'append_cons_hint' uses sorry
 bad_unification_hint.lean:12:0: error: invalid unification hint, failed to unify constraint #1
+bad_unification_hint.lean:12:0: warning: declaration 'cons_append_hint'' uses sorry
```

Comment:
There is some reordering compared to `-j0`, but it doesn't look less right in any way, and should be deterministic.